src/kOS.Safe/Execution/CPU.cs

```diff
@@ -1074,19 +1115,16 @@ public void KOSFixedUpdate(double deltaTime)
                 {
                     ProcessTriggers();
 
-                    if (currentStatus == Status.Running || currentStatus == Status.Waiting)
```

Comment:
This statement was literally irrelevant anyway.  currentStatus was an enum, and it was an enum where these were the *only* two values defined for the enum.  It was impossible for this condition to ever be false.